services/permits/tests/test_parse_hierarchy.py

```diff
@@ -0,0 +1,615 @@
+            "meta": {"bounding_box": {"left": 2.0333}},
+        },
+        {
+            "text": "(ii) The Permittee must ensure all employees and contractors involved in the operation of the WRTSF are trained in the implementation of the OMS Manual and that records of the training be maintained on site.",
+            "meta": {"bounding_box": {"left": 2.0293}},
+        },
+        {
+            "text": "(iii) The Permittee must ensure that all subaqueous placement of tailings is to be performed under the guidance of a Professional Engineer.",
+            "meta": {"bounding_box": {"left": 2.0295}},
+        },
+        {
+            "text": "(iv) The Permittee must assign a qualified person to monitor dumping and to provide a daily work schedule that shows the locations and quantities of waste rock to dump. No dumping of material must occur unless approved by the qualified person.",
+            "meta": {"bounding_box": {"left": 2.0295}},
+        },
+        {
+            "text": "(v) The Permittee must ensure that no weak, cohesive materials or snow be dumped within the ultimate design limits of the subaqueous waste rock dump area, with the exception that high fines (cohesive) materials may be placed under the direction of the qualified person in areas approved by the Engineer of Record. The Permittee must ensure the locations, total volume and percentage (with respect to total placed) material of the high fines content is documented and included in the semi-annual and annual reports.",
+            "meta": {"bounding_box": {"left": 2.0235}},
+        },
+        {
+            "text": "(vi) The Permittee must ensure any weak, cohesive materials or snow that is dumped outside the ultimate design limits of the WRTSF, be documented (locations and quantities) and included in the applicable semi-annual and annual reports.",
+            "meta": {"bounding_box": {"left": 2.0247}},
+        },
+        {
+            "text": "(vii) The Permittee may, following standard operating procedures set in the OMS manual, and as outlined in Document 7.2, use a conveyor, excavator, or remotely operated vehicle to place waste rock into the ponded portion of the WRTSF.",
+            "meta": {"bounding_box": {"left": 2.0247}},
+        },
+        {
+            "text": "(viii) The Permittee must develop an operational setback and delineation of high-risk zone, including the following:",
+            "meta": {"bounding_box": {"left": 2.0199}},
+        },
+        {
+            "text": "(a) The operational setback distance at the crest is defined with a Factor of Safety FoS > 1.1 for short term analysis of deep-seated failures and areas with less than a FoS of 1.1 are identifiable as a high-risk zone;",
+            "meta": {"bounding_box": {"left": 2.4115}},
+        },
```

Comment:
This is an example from a permit that contains the following structure:
```
Section (Uppercase letter) -> Paragraph (number) -> Subparagraph (lowercase letter) -> Clause (roman numeral) -> Subclause (lowercase letter)
```
It serves as a good test case as lowercase letters are in this cased used for both subparagraphs and subclauses